app.py

```diff
@@ -275,6 +286,7 @@ class RunOptimizationReturn(NamedTuple):
     Output("wall-clock-time-quantum", "children"),
     Output("num-locations", "children"),
     Output("vehicles-deployed", "children"),
+    background=True,
```

Comment:
👍  This was what I was thinking about. Not sure if there's actually a difference between this and a `long_callback` though.

Reply:
I don't think there's any difference I think this is just what Dash recommends using now instead of long_callback